packages/default/scss/toolbar/_layout.scss

```diff
@@ -39,6 +39,44 @@
             flex-wrap: nowrap;
         }
 
+        &.k-toolbar-scrollable {
+            flex-wrap: nowrap;
+            position: relative;
+
+            &::before {
+                content: '';
+                height: 100%;
+                aspect-ratio: 1;
+                position: absolute;
+                inset-inline-start: 0;
+                z-index: 3;
+            }
+
+            &::after {
+                content: '';
+                height: 100%;
+                aspect-ratio: 1;
+                position: absolute;
+                inset-inline-end: 0;
+                z-index: 3;
+            }
+
+            &.k-toolbar-scrollable-start::before {
+                display: none;
+            }
+
+            &.k-toolbar-scrollable-end::after {
+                display: none;
+            }
+        }
+
+        &.k-toolbar-scrollable-buttons {
```

Comment:
Maybe I am misunderstanding but doesn't this 

> Add class k-toolbar-scrollable if scrolling is enabled
> Add class k-toolbar-scrollable-buttons if scrollable buttons are enabled

suggest that `k-toolbar-scrollable-buttons` should be scoped to `k-toolbar-scrollable` as well

> NB: Also applies to the Fluent theme